dbcon/execplan/simplecolumn.cpp

```diff
@@ -358,7 +316,7 @@ const string SimpleColumn::toString(bool compact) const
   output << "Column: " << data();
   datatypes::Charset cs(fResultType.charsetNumber);
   output << endl
-         << "Info: " << schemaName() << "." << tableName() << "." << columnName()
+         << "Info: " << schemaName() << "." << tableName() << "." << columnName() << ", table alias " << tableAlias()
```

Comment:
Can you represent ", table alias " in a more compressed way, e.g. alias after tableName() in parenthesis? 
```suggestion
         << "Info: " << schemaName() << "." << tableName() 
         << "(" << tableAlias << ")" 
          << "."  << columnName() << ", table alias " << tableAlias()
```

Reply:
When the code prints out a column(and this happens a lot when debugging) for every column there is an extra ` table alias ` line added. This is a cognitive waste and might make formatting even less readable b/c the column description line becomes even wider than originally.